catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java

```diff
@@ -242,24 +280,25 @@ public void shutdown() {
     LOGGER.trace("Shutting down stream processor.");
     packetBuffer.cancelTimer();
 
-    Security security = Security.getInstance();
+    Subject localSubject = null;
+    try {
+      localSubject = getSecuritySubject("127.0.0.1");
```

Comment:
Why `127.0.0.1` and not `org.codice.ddf.system.hostname`

Reply:
We're looking for the local IP, not the hostname.